src/sys/unix/uds/listener.rs

```diff
@@ -77,11 +79,7 @@ pub(crate) fn accept(listener: &net::UnixListener) -> io::Result<(UnixStream, So
         // Ensure the socket is closed if either of the `fcntl` calls
         // error below.
         let s = unsafe { net::UnixStream::from_raw_fd(socket) };
-        syscall!(fcntl(socket, libc::F_SETFD, libc::FD_CLOEXEC))?;
-
-        // See https://github.com/tokio-rs/mio/issues/1450
-        #[cfg(all(target_arch = "x86", target_os = "android"))]
-        syscall!(fcntl(socket, libc::F_SETFL, libc::O_NONBLOCK))?;
+        syscall!(fcntl(socket, libc::F_SETFD, libc::FD_CLOEXEC | libc::O_NONBLOCK))?;
```

Comment:
This uses `F_SETFD` and `F_SETFL` (`FD` vs `FL`), two different commands. So this doesn't work. See https://man7.org/linux/man-pages/man2/fcntl.2.html.